_episodes/00-setting-the-scene.md

```diff
@@ -45,15 +49,15 @@ collaborative development,
 - **virtual environments** for isolating a project from other projects developed on the same machine, and 
 - **Python code style guide** (PEP8) to make sure our code is documented, readable and consistently formatted.
 2. Once we know our way around different code development tools, techniques and conventions, in the second section we learn:
-- how to set up a **test framework** and write tests to verify the correct behaviour of the code, 
+- how to set up a **test framework** and write tests to verify the correct behaviour of the code, and 
 - how to automate and scale testing with **Continuous Integration (CI)** using 
-- **GitHub Actions** (a CI service available on GitHub) which enables automated testing on different target user platforms.
+**GitHub Actions** (a CI service available on GitHub).
 3. In the third section, we step away from writing code for a bit to look into different **software design paradigms** 
 to understand the wider landscape in which you can design software and to help you with the 
 design decisions you have to make. Each paradigm represents a slightly different way of thinking about and structuring 
 the code and each has certain strengths and weaknesses when used to solve particular types of problems - 
 useful to know so you can recognise where it might be useful to switch in your own code. 
-4. Finally, in the forth section, we are back to practical applications and learning to **publish and release 
+4. Finally, we are back to practical applications and learning to **publish and release 
```

Comment:
I think "practical applications" isn't quite right, since section 3 is also practical, but I know what you mean! How about "Finally, we explore what we should consider when we **publish and release software ..."?